packages/gitbook/src/app/(site)/(content)/[[...pathname]]/page.tsx

```diff
@@ -183,6 +180,16 @@ export async function generateMetadata(props: PageProps): Promise<Metadata> {
     };
 }
 
+/**
+ * Check if the navigation is in-app, meaning the user clicks on a link.
+ */
+async function checkIsInAppNavigation() {
+    const headerList = await headers();
+    const fetchMode = headerList.get('sec-fetch-mode');
+
+    return fetchMode === 'cors';
```

Comment:
There is no other internal next headers sent that we could check to avoid using a web standard one for this?

Reply:
Nop there's nothing.